src/components/common/Stepper.vue

```diff
@@ -78,7 +78,6 @@ export default class Stepper extends Vue {
   @Getter(useStore) isAffidavitValid!: boolean
   @Getter(useStore) isAmalgamationInformationValid!: boolean
   @Getter(useStore) isBusySaving!: boolean
-  @Getter(useStore) isContinuationInBusinessBcValid!: boolean
```

Comment:
Since you've obsoleted this getter (and this was the only place it was used), please remove it from the store.

I think you can also delete `setContinuationInBusinessBcValid()` from the store, and also the respective state variable (see continuation-in-state-interface.ts and state-model.ts).

Reply:
Removed the unused getter and variables